Dockerfile

```diff
@@ -27,8 +36,8 @@ RUN mkdir -p /var/parking-service/static
 
 # Clean up
 USER root
-RUN yum clean all
-RUN rm -rf /var/cache/yum
+RUN dnf clean all
+RUN rm -rf /var/cache/dnf
```

Comment:
I think these should be put under the same `RUN` which installs and removes things i.e. the `RUN` starting at line 18.

Reply:
Changed.